### DISCUSSION TITLE
Pixi3D - a 3D rendering library build on top of PixiJS

### DISCUSSION THREAD

**Discussion by jnsmalm** [pixijs/pixijs#8190] [Show and tell]
Hello!

I just want to show a 3D rendering library I have been working on for quite some time (maybe you have already seen it). It's built on top of PixiJS and have seamless integration with 2D applications. It's production ready and have been used in several real world projects. Here is an motorbike which has been rendered using Pixi3D. PixiJS can sure produce some cool looking images with a glTF-loader and some shaders on-top ;-)

If you are interested and want to know more, visit https://github.com/jnsmalm/pixi3d

![spy-hypersport](https://user-images.githubusercontent.com/10713751/154327836-b7fc57da-1acd-49d1-a561-737bc4de389d.png)

The model itself was downloaded from https://skfb.ly/o8z7t

**Comment by lvanngo** [comment]
Do you guys recommend using pixi3D for something like this then https://observablehq.com/@rkaravia/2d-shapes-in-three-js?  I was hoping to get an answer on whether I can do down the path of pixi-projection and how to ask for Graphics support before.

**Reply by jnsmalm** [reply]
This can be done using Pixi3D, but not sure I recommend it. Pixi3D really doesn't have any concept of shapes so you would have to create the geometry manually.

**Reply by lvanngo** [reply]
OK Thanks for your answer @jnsmalm ...  @ivanpopelyshev it would be great if we can get graphics support into pixi projection?

**Comment by goldenratio** [comment]
@jnsmalm  thanks for the awesome work. We are already using it in our games.

**Reply by jnsmalm** [reply]
It would be interesting to take a look at something if you want to share.

**Reply by goldenratio** [reply]
they are still in progress. Will share soon.
